### PR TITLE
Updated 13.6 redirects on transition

### PR DESCRIPTION
**Pull request by kathycarothers**
Please see Google redirect document

### CODE REVIEW

File: redirects-transition.conf [hunk 1]
```diff
@@ -529,6 +542,24 @@ rewrite ^/law/policy/internet09/sunlighthearingcomment.pdf https://www.fec.gov/r
...
+# law/policy/nationwideiereporting redirects 
+rewrite ^/law/policy/nationwideiereporting/draftnationwideiereporting.pdf https://www.fec.gov/resources/cms-content/documents/draftnationwideiereporting.pdf redirect;
+
+# law/policy/probablecause/ redirects 
+rewrite ^/law/policy/probablecause/notice_2006-19.pdf https://www.fec.gov/resources/cms-content/documents/notice_2006-19.pdf redirect;
+rewrite ^/law/policy/probablecause/comm02.pdf https://www.fec.gov/resources/cms-content/documents/probablecause_comm02.pdf redirect;
+rewrite ^/law/policy/probablecause/comm01.pdf https://www.fec.gov/resources/cms-content/documents/probablecause_comm01.pdf redirect;
+rewrite ^/law/policy/probablecause/comm03.pdf https://www.fec.gov/resources/cms-content/documents/probablecause_comm03.pdf redirect;
+rewrite ^/law/policy/probablecause/comm04.pdf https://www.fec.gov/resources/cms-content/documents/probablecause_comm4.pdf redirect;
+
```
Comment (dorothyyeager) [inline redirects-transition.conf:554]: Check spacing - kind of weird looking line break in lines 550-553

File: redirects-transition.conf [hunk 2]
```diff
@@ -483,7 +489,13 @@ rewrite ^/law/litigation/citizens_guide_brochure.pdf https://www.fec.gov/introdu
 
 # law/policy/ redirects
 rewrite ^/law/policy/notice_2006-11.pdf https://www.fec.gov/resources/cms-content/documents/fedreg_notice_2006-11_EO13892.pdf redirect;
-rewrite ^/law/policy/purposeofdisbursement/notice_2006-23.pdf https://www.fec.gov/resources/cms-content/documents/fedreg_notice_2006-23_EO13892.pdf redirect;
+rewrite ^/law/policy/disbursement_purpose.pdf https://www.fec.gov/resources/cms-content/documents/disbursement_purpose.pdf redirect;
+rewrite ^/law/policy/2004/notice2004-20.pdf https://www.fec.gov/resources/cms-content/documents/notice2004-20.pdf redirect;
+rewrite ^/law/policy/iedisseminationdate/seiucomment.pdf https://www.fec.gov/resources/cms-content/documents/seiucomment.pdf redirect;
+rewrite ^/law/policy/iedisseminationdate/pomeranzcomment.pdf https://www.fec.gov/resources/cms-content/documents/pomeranzcomment.pdf redirect;
+rewrite ^/law/policy/enforcement_disclosure/perkins_coie.pdf https://www.fec.gov/resources/cms-content/documents/perkins_coie.pdf redirect;
```
Comment (dorothyyeager) [inline redirects-transition.conf:496]: Need to move lines 496-498 above line 494 (for alphabetizing the entries)

File: redirects-transition.conf [hunk 3]
```diff
@@ -529,6 +542,24 @@ rewrite ^/law/policy/internet09/sunlighthearingcomment.pdf https://www.fec.gov/r
 rewrite ^/law/policy/internet09/webmanageremails.pdf https://www.fec.gov/resources/cms-content/documents/webmanageremails.pdf redirect;
 rewrite ^/law/policy/internet09/westcomments072909.pdf https://www.fec.gov/resources/cms-content/documents/7-29-09websitecomments.pdf redirect;
 
+# law/policy/nationwideiereporting redirects 
+rewrite ^/law/policy/nationwideiereporting/draftnationwideiereporting.pdf https://www.fec.gov/resources/cms-content/documents/draftnationwideiereporting.pdf redirect;
```
Comment (dorothyyeager) [inline redirects-transition.conf:546]: Check spacing on line 546

**Review by dorothyyeager — COMMENTED** [review]
@kathycarothers Looks good overall. There were a couple of places where the break in the URL seemed a little strange so have flagged those for you to doublecheck for extra spaces and if needed repaste as plain text.

File: redirects-transition.conf (hunk 3 above)
Comment (kathycarothers) [reply redirects-transition.conf:547]: I went back and stuck the changes into notepad to remove the formatting

**Review by dorothyyeager — CHANGES_REQUESTED** [review]
Caught one more thing - delete redirects on lines 250 and 253 since your new broader redirect will cover them.

Thanks!

**Comment by kathycarothers** [comment]
Good catch for lines 250 and 253. I have deleted them.

**Review by dorothyyeager — APPROVED** [review]
Looks good; thanks @kathycarothers

**Review by patphongs — APPROVED** [review]
Thanks @kathycarothers, looks good! I have also added the ongoing court case and selected court case pages for redirects as we have already decided this when we met.

**Pull request merged**